tests/lib/rules/no-classic-classes.js

```diff
@@ -104,5 +104,33 @@ ruleTester.run('no-classic-classes', rule, {
       output: null,
       errors: [{ message: ERROR_MESSAGE, line: 3, type: 'CallExpression' }],
     },
+    {
+      code: `
+        import CustomClass from 'my-custom-addon';
+        export default CustomClass.extend({});
+      `,
+      options: [
+        {
+          additionalInvalidImports: ['my-custom-addon'],
+        },
+      ],
+      output: null,
+      errors: [{ message: ERROR_MESSAGE, line: 3, type: 'CallExpression' }],
+    },
+    {
+      code: `
+        import CustomClass from 'my-custom-addon';
+        const CustomSub = CustomClass.extend({});
```

Comment:
I think we can omit this second test case as being redundant. It doesn't give us any additional coverage.

Reply:
I was more adding it to ensure this works for this use case, but you're right in that it doesn't give us any more coverage. Removing.

Reply:
I still see it.

Reply:
Gone now.